src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/AbstractTask.groovy

```diff
@@ -158,10 +158,10 @@ abstract class AbstractTask extends DefaultTask {
      */
     protected void copyConfigFiles() throws IOException {
```

Comment:
In ci.maven, copyConfigFiles() is called in multiple tasks and not just libertyCreate.  You can either address it here or open a separate issue to handle it.

CreateServerMojo   -> libertyCreate
DebugServerMojo    -> libertyStart
DumpServerMojo     -> libertyDump
JavaDumpServerMojo -> libertyJavaDump
PackageServerMojo  -> libertyPacakge
RunServerMojo      -> libertyRun
StartServerMojo    -> libertyStart

installAppsMojo    -> not available in ci.gradle yet.